sw/airborne/arch/stm32/subsystems/radio_control/spektrum_arch.c

```diff
@@ -47,7 +47,7 @@
 #define ONE_MHZ 1000000
 
 /* Number of low pulses sent to satellite receivers */
-#define MASTER_RECEIVER_PULSES 5
+#define MASTER_RECEIVER_PULSES 4     // RAB : Changed from 5 to 4 to enable the R100 satellite rx to bind successfully
```

Comment:
What's the best way to "split" this out so that it gets pulled into `master`? This is some code that didn't get pushed as part of a prior pull. Mostly due to my inexperience with git. Thanks.